stdlib/distutils/command/__init__.pyi

```diff
@@ -0,0 +1,48 @@
+import sys
+
+from . import (
```

Comment:
What is the purpose of this import?

Reply:
Otherwise the names in `__all__` are seen as undefined.